discovery-provider/src/utils/config.py

```diff
@@ -124,3 +124,18 @@ def _load_item(self, section_name, key):
     Missing delegate owner wallet ({owner_wallet}) and/or delgate private key ({private_key}): {e}
     """
     ) from e
+
+try:
+    # get latitude longitude
+    ip_info_url = "https://ipinfo.io"
+    ip_info_response = requests.get(ip_info_url)
+    latitude, longitude = ip_info_response.json()["loc"].split(",")
+    shared_config["serviceLocation"]["serviceLatitude"] = latitude
+    shared_config["serviceLocation"]["serviceLongitude"] = longitude
+
+except (KeyError, RuntimeError) as e:
+    raise RuntimeError(
+        f"""
+    Failed to get latitude ({latitude}) and/or longitude ({longitude}): {e}
```

Comment:
I think we probably shouldn't block startup here (just in case ipinfo rate limits us or fails to return). probably worth just swallowing the error and logging it near-term. we can observe if we have failures in prod and hunt those providers down

Reply:
Yup makes sense, I'll make the change.